src/site/_includes/partials/navigation-drawer-course.njk

```diff
@@ -16,7 +16,8 @@
       </div>
 
       <div class="drawer-course__search">
-        <div class="input"></div>
+        <web-search resultsId="search-course-results" tag="{{ searchTag }}"></web-search>
+        <web-search-results id="search-course-results"></web-search-results>
```

Comment:
I wonder if we should exclude the courses stuff so it doesn't block landing the PR. I think we may need to do some animation and whatnot to fade in the search results and include some other stuff to match Jiwoong's mocks and we could do all of that in a separate PR so we can land this.